src/mono/wasm/test-main.js

```diff
@@ -509,6 +540,26 @@ const App = {
             console.error("exception thrown in", fqn);
             throw exc;
         }
+    },
+
+    create_function(...args) {
```

Comment:
unused right now? just checking. You don't have to remove it if it is going to be used in the upcoming PR.

Reply:
right, one of the next PRs will use it.